app/platform/fabric/sync/SyncService.ts

```diff
@@ -375,25 +383,63 @@ export class SyncServices {
 		this.synchInProcess.push(synch_key);
 
 		const channel_genesis_hash = client.getChannelGenHash(channel_name);
-		const blockHeight = parseInt(channelInfo.height.low) - 1;
-		// Query missing blocks from DB
-		const results = await this.persistence
+		const latestBlockHeight = parseInt(channelInfo.height.low) - 1;
+
+		// Get the value of noOfBlocks from configuration file
+		const noOfBlocks = network_configs[network_id].noOfBlocks;
+
+		// Calculate the starting block height for sync
+		const startingBlockHeight = Math.max(0, latestBlockHeight - noOfBlocks + 1);
+
+		// Syncing Details
+		logger.info(
+			`Syncing blocks from ${startingBlockHeight} to ${latestBlockHeight}`
```

Comment:
Please see the condition when number of blocks in config.json is greater than the number of blocks in ledger.

Reply:
This has been handled. if the blocks that are mentioned in the config.json file, is greater than the blocks available, it will sync the blocks that are available at that point of time.